### ISSUE TITLE
Some javascript errors 

### ISSUE THREAD

**Issue by elshaddae** [vitorfs/bootcamp#176]
Hello, please help me with some js/websockets issues, 
1. channels is not defined in the static/js/bootcamp.js line 95: `let webSocket = new channels.WebSocketBridge();`
Also, the path static/channels/js/websocketbridge.js doesn't exist.

**Comment by sebastian-code** [comment]
Hi @elshaddae. Please verify than you started the REDIS server, and check than you provided the proper configurations and environment variables.

**Comment by elkd** [comment]
@sebastian-code thank you for your response. I have failed to solve these issues, and I think maybe the code is wrong. Inside static, there is no "visible" file called channels, or do I have to add it by myself ?! the same for the channels.WebSocketBridge, it is not defined.

Also, forgive me for introducing many things in one issue, as I think they are small things that you can just help me with one/two comments. REDIS server is running and I have well defined env variables.

**Comment by sebastian-code** [comment]
I see you're using Docker @elshaddae, before going further I want to ask you some questions:
- Did you used the `docker-compose` files provided? 
- Which one?
- Did you checked the wiki provided with this project?

**Comment by elshaddae** [comment]
I am using docker-compose files yes. local.yml 
I have read the wiki also cookie-cutter, redis documentation.
I am using redis 2.10.6 because version 3 throws cacheKey error

**Comment by sebastian-code** [comment]
Hi @elshaddae I'll check, because apparently is not loading the django-channels package.

**Comment by elshaddae** [comment]
Hello @sebastian-code any update on Django-channels issue, I still can't find the solution.

**Comment by sebastian-code** [comment]
Hi @elshaddae I already checked the issue and the problem is related to the way the containers are being created when you use the `local.yml` or the `production.yml` _docker-compose_ files. I'll try to workout a solution.

**Comment by sebastian-code** [comment]
Hi @elshaddae was this helpful to you?

**Comment by elshaddae** [comment]
@sebastian-code, thank you for the info, I will figure out the solution, meanwhile, you can close the issue, it can be reopened when I (others) have found the solution or at least face the similar issue

**Comment by elshaddae** [comment]
@sebastian-code, I realized the problem is totally not related to the docker container. I tried the dev.yml but the issue was there I even tried running everything locally including redis. 
I realized the channels javascript WebSocket wrapper [here](https://www.tutorialdocs.com/tutorial/django-channels/websocke-wrapper.html), was removed in channels 2.1.4 [commit](https://github.com/django/channels/commit/2a9d764ad03927581aa2bfcadccc3e953949cb98#diff-23ae28c846e3ac6f67a8baf687f9eb36). Previously, I could find this info in the release notes 2.1.4 but now surprisingly,  it has been edited/removed, making as if js wrapper never existed in channels project. They are encouraging users to implement their own javascript, but as for me i just copied the static/channels/js/websocketbridge.js to my local files, and everything works
